### ISSUE TITLE
Learning implications for loss_scale

### ISSUE THREAD

**Issue by daegonYu** [McGill-NLP/llm2vec#110]
hello!

The loss scale is set to 20 for sentence similarity learning in SimCSE and 50 (default) for supervised contrastive training. Are there any benefits resulting from this loss scale change?

**Comment by vaibhavad** [comment]
Hi @daegonYu,

We did not run ablations on the choice of loss scale. We followed existing literature on this - [Echo embeddings ](https://github.com/jakespringer/echo-embeddings) hyperparameters for sentence similarity and [SimCSE](https://github.com/princeton-nlp/SimCSE) for unsupervised contrastive learning.

**Comment by daegonYu** [comment]
oh! I misunderstood. thank you for telling me!
